modules/ROOT/pages/comments-using-comments.adoc

```diff
@@ -1,56 +1,77 @@
-= Using TinyMCE Comments
+= Using {productname} {pluginname}
 :navtitle: Using Comments
-:description: How to add, edit, resolve, and remove comments in TinyMCE
+:description: How to add, edit, resolve, and remove comments in {productname}
 :keywords: comments, commenting, tinycomments
 :pluginname: Comments
 :plugincode: comments
 
-I'm trying to:
+== Overview
 
-* xref:add-a-comment[Add a comment].
-* xref:edit-a-comment[Edit a comment].
-* xref:delete-a-comment[Delete a comment].
-* xref:delete-a-comment-thread-conversation[Delete a comment thread (conversation)].
-* xref:resolve-a-comment-thread-conversation[Resolve a comment thread (conversation)].
-* xref:show-or-view-a-comment[Show or view a comment].
-* xref:delete-all-comment-threads[Delete all comment threads].
+This guide provides comprehensive instructions for managing comments, including how to xref:add-a-comment[add comments] to selected content, xref:edit-a-comment[edit existing comments], and xref:delete-a-comment[delete individual comments] or xref:delete-a-comment-thread-conversation[entire conversation threads]. It explains the process of xref:resolve-a-comment-thread-conversation[resolving comment threads] and xref:show-or-view-a-comment[viewing comments] using the sidebar. The document also covers features like xref:delete-all-comment-threads[deleting all comment threads] in a document.
```

Comment:
```suggestion
This guide provides visual instructions for using the comments plugin, including how to xref:add-a-comment[add new comments] to selected content, xref:reply-to-comment[replying to comments], xref:edit-a-comment[edit existing comments], and xref:delete-a-comment[delete individual comments] or xref:delete-a-comment-thread-conversation[entire conversations]. It also covers the process of xref:resolve-a-comment-thread-conversation[resolving conversations], xref:show-or-view-a-comment[viewing comments] using the sidebar, and xref:delete-all-comment-threads[deleting all conversations] in a document.
```
I think it might be worth adding a guide on replying to comments too, now that it's a little more complicated. Also, opening a conversation with multiple replies, and showing and navigating all the current conversations in the sidebar.

Reply:
@MitchC1999 - `reply-to-comment` doesnt exist as such, as we only have `Add comment...` when a conversation or card is focused, so adding a guide to this might be considered a duplicate IMO.

In regards to adding documentation about keyboard navigation, tabbing into or selecting cards..., at this stage this is something I have been advised we will not be adding.

Let me know your thoughts before i resolve this thread.

Reply:
Added guide to reply to comment [in this commit](https://github.com/tinymce/tinymce-docs/pull/3457/commits/056d29e20da063b265d802338f685aa8e5955691)